opentreemap/treemap/audit.py

```diff
@@ -790,6 +790,11 @@ def __init__(self, *args, **kwargs):
 
         self._has_been_masked = False
 
+        # When any other field is writable, any field in `_joint_writable`
+        # is writable.
+        # `_joint_writable` fields are always readable.
+        self._joint_writable = {'id'}
```

Comment:
I think we should call this field `_always_writable` instead of `_joint_writable`, and remove the first two comment lines.